cpg_workflows/inputs.py

```diff
@@ -26,20 +26,41 @@ def create_cohort() -> Cohort:
     """
     Add datasets in the cohort. There exists only one cohort for the workflow run.
     """
-    analysis_dataset_name = get_config()['workflow']['dataset']
-    dataset_names = get_config()['workflow'].get('input_datasets', [analysis_dataset_name])
-    skip_datasets = get_config()['workflow'].get('skip_datasets', [])
+    config = get_config()['workflow']
+    analysis_dataset_name = config['dataset']
+    custom_cohort_ids = config.get('input_cohorts', [])
+    input_datasets = config.get('input_datasets', [])
+    skip_datasets = config.get('skip_datasets', [])
+
+    # Additional logic to support cohorts + datasets as inputs. In future, cohorts will only be supported.
+    if custom_cohort_ids and input_datasets:
+        raise ValueError('Cannot use both custom_cohort_ids and input_datasets in the same workflow')
+
+    if custom_cohort_ids:
+        # TODO: Handle more than one cohort
+        sgs_by_dataset = get_metamist().get_sgs_by_project_from_cohort(custom_cohort_ids[0])
```

Comment:
assuming this is still a placeholder until multi-cohort runs exist? It would be good to have the process fail out if `custom_cohort_ids` is longer than 1 entry until this is supported